bigtable_automation/java/dataflow/src/main/java/org/datacommons/dataflow/CloudBigtableOptions.java

```diff
@@ -28,4 +28,10 @@ public interface CloudBigtableOptions extends DataflowPipelineOptions {
 
   @SuppressWarnings("unused")
   void setBigtableAppProfileId(ValueProvider<String> bigtableAppProfileId);
+
+  @Description("The maximum number of workers for Dataflow jobs.")
+  ValueProvider<String> getBigtableMaxNumWorkers();
```

Comment:
Is there a way to specify defaults?

Reply:
I think the default behavior in this case is to use the Dataflow autoscaling